app/controllers/course/admin/assessment_settings_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Course::Admin::AssessmentSettingsController < Course::Admin::Controller
+  add_breadcrumb :index, :course_admin_assessments_path
+
+  def index
+  end
+
+  def update
+    if current_course.update_attributes(category_params)
+      redirect_to(course_admin_assessments_path, success: t('.success'))
+    else
+      redirect_to(course_admin_assessments_path, danger: t('.failure'))
```

Comment:
this should be render 'edit' or render 'index' or something to that effect, because if it fails the user would lose all his config.